src/ucp/api/ucp_def.h

```diff
@@ -744,6 +744,15 @@ typedef struct ucp_ep_params {
      */
     ucp_conn_request_h      conn_request;
 
+    /**
+     * Tracing and analysis tools can identify the endpoint using this name.
```

Comment:
Endpoint name. Tracing and analysis tools can identify the endpoint using this name.